containers/shop/product-modal.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <div :class="{'modal-open': open}" class="modal-overlay column">
+    <BalanceTooltip />
+    <ProductCard v-bind="product" closable />
```

Comment:
Can it close on clicking outside?
Also, the close button can hide behind the header, and would even be unclickable if the screen height is shorter than product modal
![image](https://user-images.githubusercontent.com/11016151/62326123-a8739600-b4ad-11e9-81fa-d05fc9d2d1fb.png)

Reply:
It would be fixed after some markup changes. About click-away: we will do it for modals, dropdowns and dialogs after some time. Right now 'X' button is all right